Assets/Scripts/Source/Game/GameFlowManager.cs

```diff
@@ -1,6 +1,40 @@
 using UnityEngine;
-
+using UnityEngine.SceneManagement;
 public class GameFlowManager : MonoBehaviour 
 {
-    // TODO
+
+    public GameObject combatCanvas;
+    public GameObject combatUI;
+    public GameObject camera;
+
+    public static GameFlowManager instance = null;
+
+    private void Awake()
+    {
+        if (instance == null)
+            instance = this;
+
+        else if (instance != this)
+            Destroy(gameObject);
```

Comment:
Alternatively to the `if` block comment, just add a `return` after `Destroy` so we don't run the rest of this logic